### PR TITLE
Add 'medium' VM size to VM Instance Size.

### PR DESCRIPTION
**Pull request by qasim-at-tci**
Please review.
Thanks.

### CODE REVIEW

**Comment by arturgrigorenko201** [comment]
gh pr checkout 3128

**Review by arturgrigorenko201 — APPROVED** [review]

File: user/reference/overview.md [hunk 1]
```diff
@@ -240,7 +240,7 @@ If you need more than the default 2vCPU and RAM, you can run a build on a larger
 
 ```yaml
 vm:
-  size: [large|x-large|2x-large]
+  size: [medium|large|x-large|2x-large]
```
Comment (michal-at-travisci) [inline user/reference/overview.md:243]: actually if you use no `size` tag, the medium/standard is assumed by default and is not necessary to be explicitly called out (unless you want a full visibility in mixed size environment). Maybe there should be a note added about `medium` being assumed by default and thus not requiring explicit `sie: medium`?

File: user/reference/overview.md [hunk 2]
```diff
@@ -257,9 +257,10 @@ Each tier of instance size delivers more vCPU and RAM resources available at you
 
 | size      | vCPU        | Memory GiB  | Comment                                                 |
 |:---------:|:-----------:|:-----------:|:-------------------------------------------------------:|
+| medium    | 2           | ~8          | default                                                 |
```
Comment (michal-at-travisci) [inline user/reference/overview.md:260]: if you're adding `medium` here as default, please extend comment: from `default` to `default and assumed automatically if no size: medium present in the .travis.yml` or something along those lines

**Review by michal-at-travisci — CHANGES_REQUESTED** [review]
@qasim-at-tci  if we're adding the default assumed size into documentation, we need to be very explicit, that it doesn't have to be called out explicitly and is assumed automatically if size tag is not used in .travis.yml

**Pull request closed without merging**